CONTRIBUTING.md

```diff
@@ -15,39 +13,47 @@ We encourage you to read this project's CONTRIBUTING policy (you are here), its
 
 ## Getting Started
 
-<!--- TODO: If you have 'good-first-issue' or 'easy' labels for newcomers, mention them here.-->
+To run the project locally,
+
+1. Start up a HTTP server using python: `python3 -m http.server 8000`
+2. Upon launch, here is the form! Right click to inspect to open developer tools for troubleshooting
+3. Fill out form
+4. Click submit, which triggers a local download of the completed code.json
 
 ### Team Specific Guidelines
 
-<!-- TODO: This section helps contributors understand any team structure in the project (formal or informal.) Encouraged to point towards the MAINTAINERS.md file for further details.-->
+- Please try to keep pull requests to a reasonable size; try to split large contributions to multiple PRs
+- Please create pull requests into dev unless the contribution is some kind of bugfix or urgent hotfix.
+- Document and explain the contribution clearly according to provided standards when possible.
+- Feel free to reach out to us if there is any confusion. A list of the project maintainers is found here: [MAINTAINERS.md](./MAINTAINERS.md)
```

Comment:
Lines 25, 28 do not have periods, lines 26, 27 do have periods. Should these be consistent?

Reply:
Definitely, good flag